<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootDialog</class>
 <widget class="QDialog" name="RootDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawienia roota</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>reset tabel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="towar">
        <property name="text">
         <string>towarów</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="klient">
        <property name="text">
         <string>klientów</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="zapisane">
        <property name="text">
         <string>zapisanych ofert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="users">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>użytkowników</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset">
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uinit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>user init</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>baza danych</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QPushButton" name="db_new">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>utwórz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="db_import">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="db_export">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="db_del">
        <property name="text">
         <string>usuń</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="addUser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Dodaj użytkownika</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RootDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RootDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
